roles/edpm_libvirt/templates/edpm_libvirt.target

```diff
@@ -1,11 +1,11 @@
 [Unit]
-Wants=edpm_libvirt_virtlogd.service
-Wants=edpm_libvirt_virtnodedevd.service
-Wants=edpm_libvirt_virtproxyd.service
-Wants=edpm_libvirt_virtqemud.service
-Wants=edpm_libvirt_virtsecretd.service
-After=edpm_libvirt_virtlogd.service
-After=edpm_libvirt_virtnodedevd.service
-After=edpm_libvirt_virtproxyd.service
-After=edpm_libvirt_virtqemud.service
-After=edpm_libvirt_virtsecretd.service
+Wants=virtlogd.service
```

Comment:
ditto, let's use the target defined by the package.
We don't want to maintain a copy of it here